### ISSUE TITLE
Legacy print with invalid indentation produces wrong error

### ISSUE THREAD

**Issue by lysnikolaou** [we-like-parsers/cpython#112]
```
╰─ ./python.exe
Python 3.9.0a5+ (heads/pythonrun-fix-dirty:0fd6b4d7b2, Apr 21 2020, 17:17:44)
[Clang 11.0.0 (clang-1100.0.33.8)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def f():
... print "Hello, world"
  File "<stdin>", line 2
    print "Hello, world"
    ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Hello, world")?
```

This should raise an `IndentationError`.

**Comment by lysnikolaou** [comment]
I guess, this can be disabled as well for alpha6. If a user encounters this, they'll see the IndentationError right after they fix their `print` call?

**Comment by lysnikolaou** [comment]
Fixed in #111.
